src/session/sessionReducer.js

```diff
@@ -24,6 +24,16 @@ import { getHasAuth } from '../account/accountsSelectors';
  * See {@link SessionState} for discussion of what "non-persistent" means.
  */
 export type PerAccountSessionState = $ReadOnly<{
+  /**
+   * The event queue ID that we're currently polling on, if any.
```

Comment:
I like this summary line! This seems like a good basis for thinking about what the behavior should be.

Reply:
Yeah. Given that in that situation when the request eventually finishes, we're going to throw away the result without doing anything with it, I think it's still fair to say that we're not "currently polling on" that queue -- we do still have an HTTP request open, but as far as the app's logical behavior is concerned it already may as well not exist.